src/components/AddressCard.tsx

```diff
@@ -59,7 +59,7 @@ const AddressCard = ({ style, addressHash, onSettingsPress }: AddressCardProps)
 
   const [loading, setLoading] = useState(false)
 
-  const totalAddressBalance = BigInt(address?.balance ?? 0) + BigInt(address?.lockedBalance ?? 0)
+  const totalAddressBalance = BigInt(address?.balance ?? 0)
```

Comment:
I can't believe we did not notice this mistake... User reported it at #353

Reply:
Wth!